dolphinscheduler-ui/src/views/projects/workflow/definition/use-table.ts

```diff
@@ -257,6 +257,20 @@ export function useTable() {
         variables.page -= 1
       }
 
+      variables.checkedRowKeys = []
+      getTableData({
+        pageSize: variables.pageSize,
+        pageNo: variables.page,
+        searchVal: variables.searchVal
+      })
+    }).catch(() => {
+      if (
+          Math.ceil((variables.columns.length - variables.checkedRowKeys.length) / variables.pageSize) < variables.page
+          && variables.page > 1
+      ) {
+        variables.page -= 1;
+      }
+
```

Comment:
There are still some problems. 

-  Such as the current page is 1.
-  Why using columns length divide checkedRowKeys length ?
-  It's better to set a common function to handle refresh table.